newscoop/library/Newscoop/Installer/Services/UpgradeService.php

```diff
@@ -164,4 +163,23 @@ public function upgradeDatabase($versionsArray, $silent = false, $showRolls = fa
 
         return true;
     }
+
+    /**
+     * Gets default alias for the first publication
+     *
+     * @return string alias
+     */
+    public function getDefaultAlias()
```

Comment:
Shouldn't this return an array of all available and in use publications? I mean if they use different aliases shouldn't we enable api access for those as well?